LunrCoreLmdb/LunrCoreLmdb.csproj

```diff
@@ -1,18 +1,36 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <LangVersion>8</LangVersion>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <RootNamespace>Lunr</RootNamespace>
+    <Version>2.3.8.5</Version>
+    <Authors>Bertrand Le Roy</Authors>
+    <Company>Decent Consulting</Company>
+    <Description>A .NET Core, LMDB-backed port of Oliver Nightingale's lunr.js library, a lightweight full-text indexing library that is "a bit like Solr, but much smaller and not as bright." Icon adapted from https://commons.wikimedia.org/wiki/File:Internal_Structure_of_the_Moon.JPG by Iqbal Mahmud under Creative Commons Attribution Share Alike 4.0 International</Description>
```

Comment:
Let's put you as the main author for this package and remove Decent Consulting. Let's change the description to something like "LMDB storage add-on for LunrCore [...]". Also add yourself to the copyright.